src/CMakeLists.txt

```diff
@@ -1,3 +1 @@
-aux_source_directory(. APP_DIR)
-
-add_executable(texas ${APP_DIR})
+add_library(tcore OBJECT Dummy.cpp Dummy.h TexasScore.cpp TexasScore.h)
```

Comment:
add a new line: `target_include_directories(tcore PUBLIC ..)`
And suggest to rename src to tcore.

if you don't rename
use `#include <src/tcore.hpp>` in test/
after rename
`#include <tcore/tcore.hpp>`